lib/galaxy/jobs/__init__.py

```diff
@@ -1492,7 +1499,8 @@ def path_rewriter( path ):
 
         # fix permissions
         for path in [ dp.real_path for dp in self.get_mutable_output_fnames() ]:
-            util.umask_fix_perms( path, self.app.config.umask, 0o666, self.app.config.gid )
+            if os.path.exists(path):
+                util.umask_fix_perms( path, self.app.config.umask, 0o666, self.app.config.gid )
```

Comment:
The path might not exist and that might be fine. Is this a common error message you see that indicates a problem of some kind?